docs/release-notes/flink-1.11.md

```diff
@@ -0,0 +1,220 @@
+The logging properties files `log4j-yarn-session.properties` and `logback-yarn.xml` haven been renamed into `log4j-session.properties` and `logback-session.xml`.
+Moreover, `yarn-session.sh` and `kubernetes-session.sh` use these logging properties files.
+
+### State
+#### Removal of deprecated background cleanup toggle (State TTL) ([FLINK-15620](https://issues.apache.org/jira/browse/FLINK-15620))
+The `StateTtlConfig#cleanupInBackground` has been removed, because the method was deprecated and the background TTL was enabled by default in 1.10.
+
+####  Removal if deprecated option to disable TTL compaction filter ([FLINK-15621](https://issues.apache.org/jira/browse/FLINK-15621))
+The TTL compaction filter in RocksDB has been enabled in 1.10 by default and it is now always enabled in 1.11+.
+Because of that the following option and methods have been removed in 1.11: 
+- `state.backend.rocksdb.ttl.compaction.filter.enabled`
+- `StateTtlConfig#cleanupInRocksdbCompactFilter()`
+- `RocksDBStateBackend#isTtlCompactionFilterEnabled`
+- `RocksDBStateBackend#enableTtlCompactionFilter`
+- `RocksDBStateBackend#disableTtlCompactionFilter`
+- (state_backend.py) `is_ttl_compaction_filter_enabled`
+- (state_backend.py) `enable_ttl_compaction_filter`
+- (state_backend.py) `disable_ttl_compaction_filter`
+
+#### Changed argument type of StateBackendFactory#createFromConfig ([FLINK-16913](https://issues.apache.org/jira/browse/FLINK-16913))
+Starting from Flink 1.11 the `StateBackendFactory#createFromConfig` interface now takes `ReadableConfig` instead of `Configuration`. 
+A `Configuration` class is still a valid argument to that method, as it implements the ReadableConfig interface. 
+Implementors of custom `StateBackend` should adjust their implementations.
+
+### PyFlink
+#### Throw exceptions for the unsupported data types ([FLINK-16606](https://issues.apache.org/jira/browse/FLINK-16606))
+DataTypes can be configured with some parameters, e.g., precision.
+However, the precision provided by users takes no effect but a default value will always be used. 
+To avoid confusion since Flink 1.11 exceptions will be thrown if the value is not supported to make it more visible to users. 
+Changes include:
+- the precision for `TimeType` could only be `0`
+- the length for `VarBinaryType`/`VarCharType` could only be `0x7fffffff`
+- the precision/scale for `DecimalType` could only be `38`/`18`
+- the precision for `TimestampType`/`LocalZonedTimestampType` could only be `3`
+- the resolution for `DayTimeIntervalType` could only be `SECOND` and the `fractionalPrecision` could only be `3`
+- the resolution for `YearMonthIntervalType` could only be `MONTH` and the `yearPrecision` could only be `2`
+- the `CharType`/`BinaryType`/`ZonedTimestampType` is not supported
+
+### Monitoring
+#### Bla ([FLINK-](https://issues.apache.org/jira/browse/FLINK-))
```

Comment:
Yes? 😅

Reply:
Of course I forgot to delete my template line :)